src/classes/GE_PaymentServices.cls

```diff
@@ -130,14 +159,21 @@ public with sharing class GE_PaymentServices {
             generator.writeStringField('iss', 'ApiKeyManager'); // Issuer
             generator.writeNumberField('exp', getExpirationDate(1)); // Expires at
             generator.writeStringField('sub', UserInfo.getUserName()); // Subject
-            generator.writeStringField('sfdo.id', 'bees'); // Org ID
+            generator.writeStringField('sfdo.id', 'bees');
             generator.writeStringField('sfdo.product', 'CRM'); // Product
-            generator.writeStringField('sfdo.roles', 'Payments.Purchase'); // Call Role
```

Comment:
We may want to continue wiring this to "Payments.Purchase" - or something to that effect. The roles we get passed to us from elevate are technically the full set of allowed operations we can make, but the only role we need to specify here in order to make the purchase call is "Payments.Purchase".

Technically we should probably block access to the credit card widget if your configured roles *don't* include "Payments.Purchase" ... not sure how we best bridge in the interim. Technically passing in *every* role we have available is more than we need however.

Reply:
Aren't we supposed to be sending whatever values we get from Payment Services/Transport layer?  Do we know if the purchase call works if we include extra roles besides Payments.Purchase?  Maybe on their end, since they are sending multiple roles at times, they are just validating that the correct role is included in the passed in string...

Reply:
This might be worth a lengthier convo with Cody about how exactly this is intended to be used - one thing I'm beginning to understand is that the authorizer and the actual purchase endpoint are pretty strongly de-coupled, so when you present your role here, I *think* it's in a sense saying "let me through, please, I need to perform and action that requires the Payments.Purchase role" and then authorizer checks all your security stuff to see if you are in fact allowed through this particular gate. I think? We should probably ask for a more authoritative answer, but - I wouldn't hold this up based on this given the priority to have this functionality available!

Reply:
Ok, thanks for the thoughtful response!  I just tried sending multiple role values as a string, like we set up in the npsp.PS_IntegrationService_TEST.buildConfigurationParams test method.  I got an error back from the service stating "Cannot read property 'map' of undefined" - so multiple values don't seem to work anyway.  I pushed a change that hard codes this callout back to using "Payments.Purchase" for the sfdo.roles value - and updated the script in the test charter.